chromium/popup.html

```diff
@@ -50,6 +50,7 @@ <h3 i18n="chrome_experimental_rules"></h3>
 </section>
 <footer>
   <a id="whatIsThis" href="https://www.eff.org/https-everywhere" target="_blank" tabindex="-1" i18n="chrome_what_is_this">blah</a>
+  <p><small>(Version: <span id="current-version"></span>)</small></p>
```

Comment:
Please use the i18n message "about_version", which will be shared with Firefox under my pending chrome-translations branch.